### PR TITLE
fix: Issue with compiler minifying webOS device properties

### PR DESCRIPTION
**Pull request by stuartflanagan**
Fixes https://github.com/shaka-project/shaka-player/issues/6533

### CODE REVIEW

**Comment by stuartflanagan** [comment]
Fixes: #6533

**Comment by avelad** [comment]
@stuartflanagan please sign the CLA!

File: lib/util/platform.js
Comment (avelad) [inline lib/util/platform.js:685]: This is not necessary because the catch already has this information in case it fails

**Review by avelad — CHANGES_REQUESTED** [review]

**Review by tykus160 — APPROVED** [review]

File: lib/util/platform.js
Comment (stuartflanagan) [reply lib/util/platform.js:685]: You would expect the catch should have worked initially then?
I can remove just not sure what issue there is setting defaults initially?

Comment (avelad) [reply lib/util/platform.js:685]: We'll leave it at that for now.

**Comment by avelad** [comment]
@stuartflanagan Once the CLA is signed I will merge the PR.

**Review by theodab — APPROVED** [review]

**Comment by stuartflanagan** [comment]
@avelad if there is a reason I cannot sign the CLA can I pass the code onto you or another developer?

**Comment by avelad** [comment]
@stuartflanagan If you give me permission, I can do it, but if you can sign the CLA I prefer that you do it and take the authorship of this fix.

**Comment by avelad** [comment]
> Will get back to you tomorrow. Thank you for your help @avelad

I created https://github.com/shaka-project/shaka-player/pull/6558, but I'll wait until you can say your final answer before doing anything. Thanks!

**Comment by stuartflanagan** [comment]
Please go ahead and merge yours when ready. I will close mine now.

**Pull request closed without merging**
